src/requirements/data/majors/ece.ts

```diff
@@ -6,39 +6,30 @@ import {
 } from '../checkers-common';
 
 const eceRequirements: readonly CollegeOrMajorRequirement[] = [
-  {
-    name: 'ENGRD 2300',
-    description: 'ENGRD 2300',
-    source:
-      'https://www.ece.cornell.edu/ece/programs/undergraduate-programs/majors/program-requirements',
-    checker: includesWithSubRequirements(['ENGRD 2300']),
-    fulfilledBy: 'courses',
-    perSlotMinCount: [1],
-    slotNames: ['Course'],
-  },
   {
     name: 'Engineering Distribution',
-    description: 'Any ENGRD course excluding ENGRD/ECE 2100, ENGRD/ECE 2200, and ENGRD/ECE 2300.',
+    description:
+      'ECE majors must complete ECE/ENGRD 2300. ECE majors interested in computer engineering are strongly encouraged to take ECE/ENGRD 2400 as their second engineering distribution class.',
     source:
       'https://www.ece.cornell.edu/ece/programs/undergraduate-programs/majors/program-requirements',
     checker: [
       (course: Course): boolean =>
         course.subject === 'ENGRD' &&
-        !courseMatchesCodeOptions(course, ['ENGRD 2300', 'ENGRD 2100', 'ENGRD 2200']),
+        !courseMatchesCodeOptions(course, ['ENGRD 2300', 'ENGRD 2100']),
     ],
     fulfilledBy: 'courses',
     perSlotMinCount: [1],
     slotNames: ['Course'],
   },
   {
     name: 'Core Courses',
-    description: 'ECE 2100, ECE 2200 ECE 3400',
+    description: 'ECE 2100, ECE 2300, ECE 2720',
     source:
       'https://www.ece.cornell.edu/ece/programs/undergraduate-programs/majors/program-requirements',
-    checker: includesWithSubRequirements(['ECE 2100'], ['ECE 2200'], ['ECE 3400']),
+    checker: includesWithSubRequirements(['ECE 2100'], ['ECE 2300'], ['ECE 2720']),
     fulfilledBy: 'courses',
     perSlotMinCount: [1, 1, 1],
-    slotNames: ['ECE 2100', 'ECE 2200', 'ECE 3400'],
+    slotNames: ['ECE 2100', 'ECE 2320', 'ECE 2720'],
   },
   {
     name: 'Foundation Courses',
```

Comment:
Per my comment on your last PR (https://github.com/cornell-dti/course-plan/pull/600#discussion_r752596580) could you make the Advanced Programming req below double countable?